.github/workflows/rust.yml

```diff
@@ -28,11 +40,33 @@ jobs:
           sudo dpkg --add-architecture i386
           sudo apt-get update
           sudo apt-get install g++-multilib zlib1g-dev:i386 libssl-dev:i386 pkg-config:i386
+          env BYOND_MAJOR=513 BYOND_MINOR=1521 ./scripts/install_byond.sh
```

Comment:
not latest?

Reply:
I copied this from the old Travis script, not sure if there's an immediately better option.

Reply:
I mean, the TG dockerfile uses [1533](https://github.com/tgstation/tgstation/blob/master/Dockerfile) at least. 1536 would also work I presume (latest at time of comment, what goon runs).

Reply:
Either way, i'd set these higher up in a global env section